### ISSUE TITLE
Build failure: rustdesk

### ISSUE THREAD

**Issue by GGG-KILLER** [NixOS/nixpkgs#225170]
### Steps To Reproduce

1. build rustdesk

### Build log

```console
$ nix log /nix/store/jaslnyaggals8lz73dwg773x32p7s393-rustdesk-1.1.9-vendor.tar.gz.drv
@nix { "action": "setPhase", "phase": "unpackPhase" }
unpacking sources
unpacking source archive /nix/store/1fv3wq5dsxcj89bny6xx084s9k7k19iy-source
source root is source
@nix { "action": "setPhase", "phase": "patchPhase" }
patching sources
applying patch /nix/store/kgkzxcsrhh8373xwsjh2y1q60hnagwzw-cargo.patch
patching file Cargo.lock
patching file Cargo.toml
@nix { "action": "setPhase", "phase": "configurePhase" }
configuring
no configure script, doing nothing
@nix { "action": "setPhase", "phase": "buildPhase" }
building
source = "git+https://github.com/open-trade/confy#630cc28a396cb7d01eefdd9f3824486fe4d8554b"
source = "git+https://github.com/liyue201/libappindicator-rs#3763cfd629dd90050af1feafa643cbfca0bf487e"
source = "git+https://github.com/liyue201/libappindicator-rs#3763cfd629dd90050af1feafa643cbfca0bf487e"
source = "git+https://github.com/TheRadioGuy/magnum-opus#171e1d021004626f7444d1e39b98f50bc3cb2604"
source = "git+https://github.com/open-trade/parity-tokio-ipc#52515618bd30ea8101bf46f6c7835e88cec9187f"
source = "git+https://github.com/open-trade/rdev#fbbefd0b5d87095a7349965aec9ecd33de7035ac"
source = "git+https://github.com/open-trade/pulsectl#5e68f4c2b7c644fa321984688602d71e8ad0bba3"
source = "git+https://github.com/open-trade/rust-sciter?branch=dyn#82025b9ba77d5ae14543009444033036dbe25917"
source = "git+https://github.com/liyue201/systray-rs#84cca4b4171661bc6c4d1ba5aaa2320ff8e085aa"
source = "git+https://github.com/open-trade/tokio-socks#3de8300fbce37e2cdaef042e016aa95058d007cf"

ERROR: The Cargo.lock contains git dependencies

This is currently not supported in the fixed-output derivation fetcher.
Use cargoLock.lockFile / importCargoLock instead.
```

### Additional context

None

### Notify maintainers
@Leixb @marius851000 

### Metadata

```console
$ nix-shell -p nix-info --run "nix-info -m"
 - system: `"x86_64-linux"`
 - host os: `Linux 6.2.7-lqx1, NixOS, 23.05 (Stoat), 23.05.20230321.19cf008`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.13.3`
 - channels(root): `""`
 - nixpkgs: `/nix/store/2yf8ybi63pc8kjl45d2ll6cg69ch9pv4-source`
```

**Comment by ocfox** [comment]
Should be fixed in #225266, thanks if you can test it.

**Comment by GGG-KILLER** [comment]
> Should be fixed in #225266, thanks if you can test it.

@ocfox tested and it works! Will close once it gets merged lands on unstable.

https://nixpk.gs/pr-tracker.html?pr=225266
